deploy/prod/service-catalog-frontend/service-catalog-frontend-env.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: service-catalog-frontend
+  labels:
+    app: service-catalog-frontend
+spec:
+  template:
+    spec:
+      containers:
+        - name: service-catalog-frontend
+          env:
+            - name: NEXTAUTH_URL
+              value: https://tjenestekatalog.fellesdatakatalog.digdir.no
```

Comment:
@KurtStian Vi går for `https://tjenestekatalog.fellesdatakatalog.digdir.no`?

Reply:
OK. Denne har den samme oppbyggingen som URL-en som brukes for den nye begrepkatalogen. Etter hvert som vi flytter frontenden til de andre katalogene til det nye monorepo-et bør vi også endre URL-ene til disse slik at de får tilsvarende oppbygging.

Reply:
Eller skulle det vært engelsk, service-catalog?

Reply:
Denne URL vil vel kun vises for brukerne når de går inn på registreringsløsningen, og det vil vel stort sett kun være norsktalende brukere som bruker løsningen. Og registreringsløsningen støtter kun norsk - i hvert fall p.t.
Uansett bør vi bruke samme oppbygging og språk på tvers av katalogene. Og URL til den nye begrepskatalogen er på norsk.

Reply:
Vi bør holde dette likt begrepskatalog, ja. Så hvis vi endrer her så bør vi også endre der

Reply:
Siden dette er en løsning for norske virksomheter så tenker jeg at vi bruker norsk i URL-ene på registreringsløsningen. Vi bruker jo norsk allerede på siden "Alle kataloger" - https://**registrering**.staging.fellesdatakatalog.digdir.no/